### PR TITLE
Java: Update MaD Declarations after Triage

### PR DESCRIPTION
**Pull request by atorralba**
This PR updates the MaD declarations after a recent triage session.

TR 328

### CODE REVIEW

**Comment by kaeluka** [comment]
@atorralba do you think we can merge this so the data can be used for our evaluation? (see backref)

File: java/ql/lib/ext/jakarta.persistence.model.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["jakarta.persistence", "EntityManager", True, "createNativeQuery", "(String,Class)", "", "Argument[0]", "sql-injection", "ai-manual"]
+      - ["jakarta.persistence", "EntityManager", True, "createQuery", "(CriteriaDelete)", "", "Argument[0]", "sql-injection", "ai-manual"]
+      - ["jakarta.persistence", "Query", True, "setParameter", "(Parameter,Object)", "", "Argument[this]", "sql-injection", "ai-manual"]
```
Comment (atorralba) [inline java/ql/lib/ext/jakarta.persistence.model.yml:8]:
```suggestion
```

**Comment by atorralba** [comment]
@jcogs33 to prevent this from being a self-approve, as the other person present during the triage session, do you mind giving this a rubber stamp?

**Review by jcogs33 — APPROVED** [review]
Approving (assuming that the `Java Integration Tests Windows` failure isn't related to the PR).

Should the change note date be updated before merging?

**Comment by atorralba** [comment]
> Should the change note date be updated before merging?

I _think_ that date is only used to make the change note filename unique. So it shouldn't matter much.

**Pull request merged**